### PR TITLE
Update equipment.c Add D7 232 Bar Tank

### PR DESCRIPTION
**Pull request by gregbenson314**
D7 232 Bar tank added

D7 232 Bar are common in Europe, not just D7 300 Bar.

### Describe the pull request:

- [ ] Bug fix
- [x] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### CODE REVIEW

**Review by bstoeger — APPROVED** [review]
I'm not sure that it is smart or useful to mention line numbers in commit messages. The line numbers may change on rebasing. In fact, I might cherry-pick this into my branch in #4198.

Apart from that: LGTM.

Please don't merge yet, because it might be simpler to import that into #4198 instead of rebasing #4198.

**Comment by mikeller** [comment]
> In fact, I might cherry-pick this into my branch in #4198.

Please don't - I think it is preferable to have this pull request for a functional change (and not a cleanup really) stay around, instead of being lost in the >100 commits in #4198.
I am happy to do the rebase for this after #4198 has been merged.

**Comment by atdotde** [comment]
Admittedly, the list of pre-defined tanks is somewhat arbitrary. But you realise you can manually add any tank you like and if it is your dive log, it will be presented to you in the pull down, i.e. this PR affects only first time users of this tank size?

**Comment by gregbenson314** [comment]
> Admittedly, the list of pre-defined tanks is somewhat arbitrary. But you realise you can manually add any tank you like and if it is your dive log, it will be presented to you in the pull down, i.e. this PR affects only first time users of this tank size?

I'm primarily an Android user and what what I can see there's no way to manually add different tank sizes on Android.  

One could edit on the desktop version and then cloud sync, but that seems quite a clunky workaround. 

I'll admit I am not a programmer so adding the capability to be able to manually add tanks on Android is outside of my abilities, although that seems like the best all-round solution?

**Pull request merged**
